compiler/luci/import/src/Nodes/CircleCast.cpp

```diff
@@ -54,8 +56,18 @@ CircleNode *CircleCastGraphBuilder::build_node(const circle::OperatorT &op,
   node->x(inputs[0]);
 
   const auto *options = op.builtin_options.AsCastOptions();
-  node->in_data_type(luci_datatype(options->in_data_type));
-  node->out_data_type(luci_datatype(options->out_data_type));
+  if (options != nullptr)
+  {
+    node->in_data_type(luci_datatype(options->in_data_type));
+    node->out_data_type(luci_datatype(options->out_data_type));
+  }
+  else
+  {
+    node->in_data_type(inputs[0]->dtype());
+    node->out_data_type(loco::DataType::Unknown);
```

Comment:
Not now, but it looks better to set `node->in_data_type` and `node->out_data_type` to `loco::DataType::Unknown` as default, instead of `loco::DataType::FLOAT32`.